lib/TTT/computer_player.ex

```diff
@@ -15,7 +15,13 @@ defmodule ComputerPlayer do
   end
 
   def make_move(state) do
-    select_move(state)
-    |> State.set_move(state)
+    cond do
+      select_move(state) !== {:error, "empty error"} ->
+        select_move(state)
+        |> State.set_move(state)
+
+      select_move(state) == {:error, "empty error"} ->
```

Comment:
This came about so that the computer_player wouldn't initiate setting a move if the board had already been filled, as it resulted in {:error, "empty error"} being passed to State.set_move.